Formula/genders.rb

```diff
@@ -35,8 +35,10 @@ class Genders < Formula
   end
 
   def install
-    ENV["PYTHON"] = which("python3")
+    ENV["PYTHON"] = which("python3.10")
     system "./configure", "--prefix=#{prefix}", "--with-java-extensions=no"
+    # Run in separate steps to avoid install failure.
+    system "make"
```

Comment:
Still failed. Let's try this.
```suggestion
    ENV.deparallelize
```